### DISCUSSION TITLE
Automatically open the correct section using qa_expand_type

### DISCUSSION THREAD

**Discussion by ejmurc** [eliasmurcray/ka-notifications#15] [Ideas]
This is a new feature I plan to add soon. It will open up the correct section based on the query parameters.

**Comment by ejmurc** [comment]
It seems it is working for now, but it is not able to detect the difference between question answer and project help answer. I will leave this open for now.

**Comment by 99TheDark** [comment]
Since this is your repo, why not start this under discussions or projects?

**Comment by ejmurc** [comment]
I don't know how to
